tron-cli/src/main/scala/org/tron/cli/App.scala

```diff
@@ -15,21 +15,31 @@ object App {
     opt[String]('t', "type").action( (x, c) =>
       c.copy(peerType = x) ).text("type can be server or client")
 
-    help("help").text("How to use")
+    cmd("help").action( (_, c) => doCopy(c, HelpCommand()))
 
-    cmd("account").action( (_, c) => c.copy(command = Some(AccountCommand()))).
+    cmd("account").action( (_, c) => doCopy(c, AccountCommand())).
       text("Shows the current account")
 
-    cmd("version").action( (_, c) => c.copy(command = Some(VersionCommand()))).
+    cmd("version").action( (_, c) => doCopy(c, VersionCommand())).
       text("Shows the current version")
 
-    cmd("exit").action( (_, c) => c.copy(command = Some(ExitCommand()))).
+    cmd("exit").action( (_, c) => doCopy(c, ExitCommand())).
       text("close tron")
   }
 
+  def doCopy(config: Config, command: Command): Config = {
+    config.copy(command = Some(command))
+  }
 
   def main(args: Array[String]) = {
```

Comment:
I do not know how to handle with the double `handleArgs(args)` - the first one is for the initialize of the application and the second for user input args.

Reply:
Removed the first handleArgs, it was used for initial args that were supplied when starting the application. We don't use startup arguments for now.